### ISSUE TITLE
After upgrade to 1.2.4, many array_space* metrics becomes zero

### ISSUE THREAD

**Issue by ruey-cheng** [PureStorage-OpenConnect/pure-exporter#27]
Hi 
Thank you for fix #26 
but after upgrade to 1.2.4, many array_space* metrics becomes zero

curl "http://localhost:9491/metrics/flasharray?endpoint=192.168.1.xx&apitoken=xxx" | grep array_space
```
# HELP purefa_array_space_datareduction_ratio FlashArray overall data reduction
# TYPE purefa_array_space_datareduction_ratio gauge
purefa_array_space_datareduction_ratio{dimension=""} 0.0
# HELP purefa_array_space_capacity_bytes FlashArray overall space capacity
# TYPE purefa_array_space_capacity_bytes gauge
purefa_array_space_capacity_bytes{dimension=""} 0.0
# HELP purefa_array_space_provisioned_bytes FlashArray overall provisioned space
# TYPE purefa_array_space_provisioned_bytes gauge
purefa_array_space_provisioned_bytes{dimension=""} 0.0
# HELP purefa_array_space_used_bytes FlashArray overall used space
# TYPE purefa_array_space_used_bytes gauge
purefa_array_space_used_bytes{dimension="shared"} 0.0
purefa_array_space_used_bytes{dimension="system"} 0.0
purefa_array_space_used_bytes{dimension="volumes"} 0.0
purefa_array_space_used_bytes{dimension="snapshots"} 0.0
purefa_array_space_used_bytes{dimension="replication"} 0.0
```

**Comment by tothf** [comment]
Have the same problem. Works fine with pure-exporter:1.2.3 and Purity 5.3.10

**Comment by genegr** [comment]
Fixed in release v1.2.5
